geetools/Image.py

```diff
@@ -1315,11 +1317,27 @@ def distance(self, other: ee.image) -> ee.Image:
 
         return ee.Image(distance)
 
-    def maskCover(self) -> ee.Image:
-        """return an image with the mask cover ratio as an image property.
+    def maskCoverRegion(
+        self,
+        region: ee.Geometry,
+        scale: Optional[int] = None,
+        band: Optional[str] = None,
+        proxy_value: int = -999,
```

Comment:
I am not sure what line are you referring to 🤔 

For `region` I agree it can be the image's footprint. But I'd set to to `None` as the others.

The `scale` param is there 'cos the user might want to save some bucks and do it at a lower resolution, it defaults to the image's resolution. 

The `band` could be `0` 🤔 but I think it might be confusing for the user. GEE API does the same for `labelBand` in https://developers.google.com/earth-engine/apidocs/ee-image-reduceconnectedcomponents and other places.

The `proxy_value` has a default. BTW, should it be `proxyValue` or `proxy` or is it ok like this?

Reply:
I would do proxyValue as it can also be a `ee.Number`. I don't know why you see all this, my comment was only about `proxyValue`

Reply:
I don't know 🤷‍♂️ btw I like setting it to `None` as default and use the min value of the type of the band. I added this param 'cos in an edge case that value could be in use, and if the user is aware of that, can change it to avoid a conflic.. again, handling edge cases 😁

Reply:
I was coding it until I realized that it can mislead us, see https://code.earthengine.google.com/b17fe783bf08a081c8a56ef49e4d44f9

Reply:
ok I'm convinced